src/app/kbv/controllers/radio-question.js

```diff
@@ -0,0 +1,85 @@
+const debug = require("debug")("load-question");
+const path = require("path");
+const BaseController = require("hmpo-form-wizard").Controller;
+const presenters = require("../../../presenters");
+
+const {
+  API: {
+    PATHS: { QUESTION, ANSWER },
+  },
+} = require("../../../lib/config");
+
+class RadioQuestionController extends BaseController {
+  configure(req, res, next) {
+    req.form.options.template = path.join(
+      req.form.options.templatePath,
+      "radio-question"
+    );
+    super.configure(req, res, next);
+  }
+
+  locals(req, res, callback) {
+    super.locals(req, res, (err, locals) => {
+      if (err) {
+        return callback(err, locals);
+      }
+
+      locals.question = {
+        ...presenters.questionToRadio(req.session.question, req.translate),
+        content: presenters.questionToContent(
+          req.session.question,
+          req.translate
+        ),
+        title: presenters.taxYearToTitle(req.session.question, req.translate),
+      };
+
+      callback(null, locals);
+    });
+  }
+
+  async saveValues(req, res, callback) {
+    await super.saveValues(req, res, async (err) => {
+      if (err) {
+        return callback(err);
+      }
+
+      const userInput = req.body[req.session.question.questionKey];
+
+      try {
+        await req.axios.post(
+          ANSWER,
```

Comment:
Sorry as just realised this isn't explicit on the ticket but this is really just a routing page with no answer to submit - the answer for this question key will come from the page that follows (the tax return question pages covered by [PYIC-4653](https://govukverify.atlassian.net/browse/PYIC-4653))

I did think we might want to build these 2 tickets together so we can complete and test the full question flow for this